Control/DeepSeq.hs

```diff
@@ -105,6 +114,10 @@ import GHC.Stack ( CallStack(..) )
 import GHC.SrcLoc ( SrcLoc(..) )
 #endif
 
+#if __GLASGOW_HASKELL__ >= 708
```

Comment:
I disagree. The whole `Coercible` apparatus is very much GHC-only. I don't think pretending this is about `base` does anyone any favors.